data/jonasermert.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Jonas Ermert",
+  "bio": "Hello, my Name is Jonas. I am a Fullstack Developer working with React Native and Golang. In my free time I love to create Flutter Apps."
```

Please add the missing comma here!

```suggestion
  "bio": "Hello, my Name is Jonas. I am a Fullstack Developer working with React Native and Golang. In my free time I love to create Flutter Apps.",
```